custom_components/ramses_cc/broker.py

```diff
@@ -129,7 +124,7 @@ def _create_client(
         if not schema_is_minimal(config_schema):  # move this logic into ramses_rf?
             _LOGGER.warning("The config schema is not minimal (consider minimising it)")
 
-        if cached_schema and (merged := merge_schemas(cached_schema, config_schema)):
+        if cached_schema and (merged := merge_schemas(config_schema, cached_schema)):
```

Comment:
These were switched which I imagine was causing issues with the cached schema taking precedence over the config schema.

Reply:
Hmmm... I'll have to look at this - can't do it until after work.

If so, we should cherry-pick for **master** branch.

Reply:
can you split it out as a separate PR? SO I can cherry-pick?

Reply:
https://github.com/zxdavb/ramses_cc/pull/117